modules/consensus/accept_txntypes_test.go

```diff
@@ -0,0 +1,688 @@
+	if err != nil {
+		panic(err)
+	}
+
+	// Mine a block to close the storage proof window.
+	_, err = cst.miner.AddBlock()
+	if err != nil {
+		panic(err)
+	}
+
+	// Check that the file contract has been removed.
+	_, err = cst.cs.dbGetFileContract(fcid)
+	if err != errNilItem {
+		panic("file contract should not exist in the database")
+	}
+
+	// Check that the siafund pool has not changed.
+	postProofPool := cst.cs.dbGetSiafundPool()
+	if postProofPool.Cmp(siafundPool) != 0 {
+		panic("siafund pool should not change after submitting a storage proof")
+	}
+
+	// Check that a delayed output was created for the missed proof.
+	spoid := fcid.StorageProofOutputID(types.ProofMissed, 0)
+	dsco, err := cst.cs.dbGetDSCO(cst.cs.dbBlockHeight()+types.MaturityDelay, spoid)
+	if err != nil {
+		panic(err)
+	}
+	if dsco.UnlockHash != fc.MissedProofOutputs[0].UnlockHash {
+		panic("wrong unlock hash in dsco")
+	}
+	if dsco.Value.Cmp(fc.MissedProofOutputs[0].Value) != 0 {
+		panic("wrong sco value in dsco")
+	}
+}
+
+// TestIntegrationMissedStorageProofBlocks creates a consensus set tester and
+// uses it to call testMissedStorageProofBlocks.
+func TestIntegrationMissedStorageProofBlocks(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	cst, err := createConsensusSetTester("TestIntegrationMissedStorageProofBlocks")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer cst.closeCst()
+	cst.testMissedStorageProofBlocks()
+}
+
+// testFileContractRevision creates and revises a file contract on the
+// blockchain.
+
+// TestIntegrationFileContractRevision creates a consensus set tester and uses
+// it to call testFileContractRevision.
+
+// testSpendEmptySiafunds spends siafunds on the blockchain when the siafund
+// pool is empty.
+
+// TestIntegrationSpendEmptySiafunds creates a consensus set tester and uses it
+// to call testSpendEmptySiafunds.
+
+// testSpendSiafunds spends siafunds on the blockchain when the siafund pool is
+// not empty.
+//
+// TODO: Make sure the ClaimStart value is being set correclty.
+
+// TestIntegrationSpendSiafunds creates a consensus set tester and uses it to
+// call testSpendSiafunds.
+
+// testDelayedOutputMaturity adds blocks that result in many delayed outputs
+// maturing at the same time, verifying that bulk maturity is handled
+// correctly.
+
+// TestRegressionDelayedOutputMaturity creates a consensus set tester and uses
+// it to call testDelayedOutputMaturity. In the past, bolt's ForEach function
+// had been used incorrectly resulting in the incorrect processing of bulk
+// delayed outputs.
+
+// testFileContractMaturity adds blocks that result in many file contracts
+// being closed at the same time.
+
+// TestRegressionFileContractMaturity creates a consensus set tester and uses
+// it to call testFileContractMaturity. In the past, bolt's ForEach function
+// had been used incorrectly, resulting in the incorrect processing of bulk
+// file contracts.
+
+/// BREAK ///
+/// BREAK ///
+/// BREAK ///
```

Comment:
They are for functions I haven't written yet but would like to. They are basically TODOs, but hopefully things I will address before this PR is merged.